src/Util/Datastore.php

```diff
@@ -62,6 +62,18 @@ public function getFile(string $filePath): array
 
         return $resource;
     }
+    
+    /**
+     * Queries Flysys for a file's mimetype
+     *
+     * @param string $filePath The file's path.
+     *
+     * @return string|false
+     */
+    public function getMimeType(string $filePath)
```

Comment:
return type missing

Reply:
yes, because the filesystemInterface->getMimeType() itself returns string or false. In PHP 8 we'll be able to specify union types. There are suggestions of making a return type interface to get around this now, but that's probably way overkill.